src/Operator.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react"
+
+function Operator({state})
+{
+    const [currentState, newState] = state
+    const [show, newshow] = useState('+')
+    const changeOperator = (f, a) =>
+    {
+        newState((a, b) => f)
+      //  console.log(currentState)
+        newshow(a)
+    }
+    return (
+        <div className="panel">
+        <p>{show}</p>
+        <div className="numbers">
+        <button onClick={() => changeOperator((a,b) => Number(a) + Number(b), '+')}>+</button>
+        <button onClick={() => changeOperator((a,b) => a - b, '-')}>-</button>
+        <button onClick={() => changeOperator((a,b) => a * b, '*')}>*</button>
```

Comment:
overly complicated to define the operator function inline here, better to define them where you compute the total and simply store the operator as a string  "-", "+" etc...